api/api_delete.py

```diff
@@ -5,62 +5,43 @@
 from flask import jsonify
 
 from utilities.db_util import connect_execute_query
+from utilities.token_utilities import owner_token_required
 
+from classes.Song import Song
 
 api_delete = Blueprint('api_delete', __name__)
 
-"""
-@api_delete.route('api/songs')
-def delete_song(song)
-    return _handle_request() #sta treba vracati?
 
-def _handle_request(wanted_params: str, endpoint: str):
-    logging.info(f"User requested delete of {endpoint}.")
-    try:
-        if invalid_parameters(wanted_params):
-            logging.info("Bad parameter added to request.")
-            return make_response(f"Bad parameter", 420)
-
-        if invalid_param_value():
-            logging.info("Bad parameter values added to request.")
-            return make_response("Bad parameter value", 421)
-
-        where_query = ''
-        for param in wanted_params:
-            if param == 'sort':
-                continue
+@api_delete.route('/api/songs',methods=['DELETE'])
+def delete_song():
+    return _handle_owner_token_request('song_name')
```

Comment:
Why are we passing here a string `'song_name'` to the request.

We need to get `request.args.get('song_name')` (check if user added that param before taking the value)